lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -774,15 +864,34 @@ for ($i = 0; $i -lt $triggers.Count; $i++) {
 
     $time_properties = @('execution_time_limit', 'delay', 'random_delay')
     foreach ($property_name in $time_properties) {
-        # validate the duration is in the Duration Data Type format
-        # PnYnMnDTnHnMnS
         if ($trigger.ContainsKey($property_name)) {
             $time_span = $trigger.$property_name
-            try {
-                [void][System.Xml.XmlConvert]::ToTimeSpan($time_span)
-            } catch [System.FormatException] {
-                Fail-Json -obj $result -message "trigger option '$property_name' must be in the XML duration format but was '$time_span'"
-            }
+            Test-XmlDurationFormat -key $property_name -value $time_span
+        }
+    }
+
+    if ($trigger.ContainsKey("repetition")) {
+        $repetition = ConvertTo-HashtableFromPsCustomObject -object $trigger.repetition
+        $trigger.repetition = $repetition
+
+        $interval_key = "interval"
```

Comment:
Just use the string `"interval"` where it is required, no need to put this in a variable that is actually longer.